applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -105,10 +116,18 @@ def GetDefaultSettings(cls):
             "line_search": false,
             "use_old_stiffness_in_first_iteration": false,
             "compute_reactions": true,
-            "block_builder" : true,
             "builder_and_solver_settings" : {
-                "diagonal_values_for_dirichlet_dofs" : "use_max_diagonal",
-                "silent_warnings"                    : false
+                "use_block_builder" : true,
+                "use_lagrange_BS"   : false,
+                "advanced_settings"                      : {
```

Comment:
for the record, I find this a good idea of separating global settings (e.g. `use_block_builder`) from detailed/advanced settings (like `silent_warnings`)
=> lets also use this for the other classes!